src/main/java/io/debezium/connector/yugabytedb/metrics/YugabyteDBStreamingPartitionMetrics.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.yugabytedb.metrics;
+
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Struct;
+
+import io.debezium.connector.common.CdcSourceTaskContext;
+import io.debezium.data.Envelope;
+import io.debezium.pipeline.meters.StreamingMeter;
+import io.debezium.pipeline.source.spi.EventMetadataProvider;
+import io.debezium.pipeline.spi.OffsetContext;
+import io.debezium.schema.DataCollectionId;
+
+/**
+ * @author Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+ */
+public class YugabyteDBStreamingPartitionMetrics extends AbstractYugabyteDBPartitionMetrics
+        implements YugabyteDBStreamingPartitionMetricsMXBean {
+
+    private final StreamingMeter streamingMeter;
+
+    public YugabyteDBStreamingPartitionMetrics(CdcSourceTaskContext taskContext,
+                                               Map<String, String> tags,
+                                               EventMetadataProvider metadataProvider) {
+        super(taskContext, tags, metadataProvider);
+        streamingMeter = new StreamingMeter(taskContext, metadataProvider);
```

Comment:
Where are these metrics set?

Reply:
The streaming meter metrics are set by some `onEvent()` function which Debezium has implemented in its core: https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/pipeline/meters/StreamingMeter.java#L69

There are other meters as well: https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/pipeline/meters/